### ISSUE TITLE
Do not suggest to use orator.py configuration file for migrations

### ISSUE THREAD

**Issue by loreilei** [sdispater/orator#243]
Hello,

I am new to Orator: I wanted to setup Orator with SQLite for a small project.
When I arrived to migration, I read in the documentation:

> For the migrations to actually work, you need a configuration file describing your databases. It can be a orator.yml file or a orator.py located where the orator command is executed, or any other yaml or python file (these must then be explicitely specified when executing commands with the --config\-c) which follow the following requirements:

I choose to use the orator.py variant. Once I set up this file, migration was working and I kept moving on my project.
However, when I launched my application, I got messages like:
> Traceback (most recent call last):
  File "test_orator.py", line 3, in <module>
    from orator import Model
ImportError: cannot import name 'Model'

So I guess this orator.py file is being considered the module file by my interpreter and that is why it didn't find the Model class in it.

A simple way to reproduce this issue is to : 
- install orator via pip,
- create an orator.py file containing a database configuration for migration
- create a file test_orator.py trying to import Model from orator
- launch the test_orator.py

Maybe this behaviour comes from my configuration (I installed module as user mode with pip, or I don't have a \__init__.py file at the root of my project).
Anyway, if this behaviour is confirmed and the configuration of python/pip is in cause, maybe it would be nice to add a few lines about it in the configuration. Or stop suggesting to use a orator.py file. Or change its default name.
Or maybe this comes from the fact that I launch orator from the root of my project (which led me to put this orator.py file at the root of my project), which might be a not recommended behaviour/configuration.

I hope this will help.

**Comment by josephmancuso** [comment]
yeah this shouldn't work lol. If you put an orator.py module in your project it will break. Not sure how that made it to the documentation and tested. Can you point where it is showing this and i'll try to get around to fixing it

**Comment by loreilei** [comment]
It took me a while to figure out why I got these import errors while it worked before my migration ^^'
I read that on this page: https://orator-orm.com/docs/0.9/migrations.html
In the paragraph where database configuration file is explained.

**Comment by jychen7** [comment]
In out project, we use `orator.py` in migrations, but we didn't use the ORM feature.
I think that's why it is good here.

**Comment by oscarmorrison** [comment]
+1 Any update on this?

I have migration working, as described here: https://orator-orm.com/docs/0.9/migrations.html

But can't get any ORM to work with the `orator.py` file there

**Comment by ilyakatz** [comment]
Same problem here, seems there is a missing piece of documentation about how to configure models with yml file

**Comment by archite** [comment]
This worked for me:

```
In [1]: from orator import DatabaseManager, Model

In [2]: import yaml

In [3]: config = yaml.safe_load(open('orator.yml'))

In [4]: db = DatabaseManager(config['databases'])

In [5]: Model.set_connection_resolver(db)
```
